x-pack/plugins/ml/public/application/jobs/jobs_list/components/datafeed_chart_flyout/datafeed_chart_flyout.tsx

```diff
@@ -154,21 +187,35 @@ export const DatafeedChartFlyout: FC<DatafeedChartFlyoutProps> = ({ jobId, end,
     setIsLoadingChartData(false);
   }, [endDate, data.bucketSpan]);
 
-  const getJobData = useCallback(async () => {
+  const getJobAndSnapshotData = useCallback(async () => {
     try {
       const job: CombinedJobWithStats = await loadFullJob(jobId);
+      const modelSnapshotResultsLine: LineAnnotationDatumWithModelSnapshot[] = [];
+      const modelSnapshotsResp = await getModelSnapshots(jobId);
+      const modelSnapshots = modelSnapshotsResp?.model_snapshots ?? [];
+      modelSnapshots.forEach((modelSnapshot) => {
+        const timestamp = Number(modelSnapshot?.latest_record_time_stamp);
+
+        modelSnapshotResultsLine.push({
+          dataValue: timestamp,
+          details: `${modelSnapshot.description}. ${revertSnapshotMessage}`,
```

Comment:
The revert snapshot message should only be appended for users with the correct capabilities.

@jgowdyelastic which is the capability(s) you check in the snapshots tab?

Reply:
Permissions check added in 69bc20da7b898b06e207d35fba64663e7612a829

Reply:
I still see the snapshot revert part of the message when logging in as ml_viewer. Clicking on the marker correctly does nothing, but would be good to hide that part of the tooltip.

Reply:
Good call - updated the tooltip in 5e9ab15b50987049abc10580bafdd45a5ee900bc